en/mkdocs.yml

```diff
@@ -328,6 +310,23 @@ nav:
         - Using the Siddhi Store REST API: develop/analytics-rest-api.md
         - Working with the Source Code: develop/working-with-the-source-code.md
         - Java Documentation: develop/java-documentation.md
+        - Extending WSO2 API Manager:
+            - Extending Key Management:
+                - develop/extending-api-manager/extending-key-management/extending-key-validation.md
+                - develop/extending-api-manager/extending-key-management/extending-scope-validation.md
+                - develop/extending-api-manager/extending-key-management/extending-the-key-manager-interface.md
+                - develop/extending-api-manager/extending-key-management/writing-custom-grant-types.md
```

Comment:
Mkdocs entry is incorrect as you have not defined the names of the menus that will appear on the left nav for these pages